packages/@vuepress/core/lib/client/components/OutboundLink.vue

```diff
@@ -1,26 +1,30 @@
 <template>
-  <svg
-    class="icon outbound"
-    xmlns="http://www.w3.org/2000/svg"
-    aria-labelledby="outbound-link-title"
-    role="img"
-    focusable="false"
-    x="0px"
-    y="0px"
-    viewBox="0 0 100 100"
-    width="15"
-    height="15"
+  <span
+    :title="openInNewWindowTitle"
+    class="sr-only"
   >
-    <title id="outbound-link-title">{{ openInNewWindowTitle }}</title>
-    <path
-      fill="currentColor"
-      d="M18.8,85.1h56l0,0c2.2,0,4-1.8,4-4v-32h-8v28h-48v-48h28v-8h-32l0,0c-2.2,0-4,1.8-4,4v56C14.8,83.3,16.6,85.1,18.8,85.1z"
-    />
-    <polygon
-      fill="currentColor"
-      points="45.7,48.7 51.3,54.3 77.2,28.5 77.2,37.2 85.2,37.2 85.2,14.9 62.8,14.9 62.8,22.9 71.5,22.9"
-    />
-  </svg>
+    <svg
+      class="icon outbound"
+      xmlns="http://www.w3.org/2000/svg"
+      aria-hidden="true"
+      role="img"
```

Comment:
I think you can remove `role="img"`

Reply:
thank you, its my first time contributing so I missed your remarks just saw that now.

Reply:
you said "I think", so I went browsing a bit:
MDN say nothing special about it but they have a side note about
**Deciding between aria-hidden="true", role="presentation", and role="none"**
Which means that to make sr ignore the SVG is should use role that is not 'img', therefore I think we can conclude that it's unnecessary.

Also the next article I references "Contextually Marking up accessible images and SVGs" doesn't use role with aria-hidden. 

So I'll remove it. 
reference:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-hidden_attribute
https://www.scottohara.me/blog/2019/05/22/contextual-images-svgs-and-a11y.html

Reply:
Sorry, I said "I think" because `role="img"` is ignored when `aria-hidden` is present, so removing it or leaving it wouldn't make much difference.

You did well to remove it.